dockstore-common/src/main/java/io/dockstore/common/yaml/YamlWorkflow.java

```diff
@@ -113,6 +116,7 @@ public void setAuthors(final List<YamlAuthor> authors) {
         this.authors = authors;
     }
 
+    @ValidTestFilesPath
     public List<String> getTestParameterFiles() {
```

Comment:
I think you can actually re-use `@ValidPrimaryPath` for `getTestParameterFiles()` too, so you don't have to create another validator that does a similar thing as `@ValidPrimaryPath`.


Renaming `@ValidPrimaryPath` to `@AbsolutePath` to make it more universal, it would look something like
```
@AbsolutePath
String getPrimaryDescriptorPath()

public List<@AbsolutePath String> getTestParameterFiles()
```

Reply:
> I think you can actually re-use `@ValidPrimaryPath` for `getTestParameterFiles()` too, so you don't have to create another validator that does a similar thing as `@ValidPrimaryPath`.
> 
> Renaming `@ValidPrimaryPath` to `@AbsolutePath` to make it more universal, it would look something like
> 
> ```
> @AbsolutePath
> String getPrimaryDescriptorPath()
> 
> public List<@AbsolutePath String> getTestParameterFiles()
> ```

Seconding this, and maybe add a `@NotNull`:
```
@NotNull
@AbsolutePath
String getPrimaryDescriptorPath()

public List<@NotNull @AbsolutePath String> getTestParameterFiles()
```

See my comment elsewhere about `null`s and `Validators`.